src/helpers/slugify.js

```diff
@@ -0,0 +1,13 @@
+const slugify = (input) => {
+  let stringVal = input.trim().toLowerCase();
+  const specialCharacters = 'àáäâèéëêìíïîòóöôùúüûñç·/_,:;';
+  const convertSpecial = 'aaaaeeeeiiiioooouuuunc------';
+  [...specialCharacters].forEach((character, index) => {
+    stringVal = stringVal.replace(new RegExp(specialCharacters.charAt(index), 'g'), convertSpecial.charAt(index));
```

Comment:
I like that you are converting these characters to normal alphabets. Really nice. Can we rename `convertSpecial` to `normalCharacters`? This would make it easier to understand what's going on.

Reply:
yes we can, I think so too